test/integrations/destinations/hs/router/data.ts

```diff
@@ -1828,4 +1913,554 @@ export const data = [
       },
     },
   },
+  {
+    name: 'hs',
+    description: 'router job ordering ',
```

Comment:
What is the exact scenario we want to test with this test case?

Reply:
With this scenario we want to test if the grouping of events is done in the right manner to maintain the event order